src/property-filter/controller.ts

```diff
@@ -1,21 +1,26 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
-import { PropertyFilterProps } from './interfaces';
-import { fireNonCancelableEvent } from '../internal/events';
+
+import { PropertyFilterProps as I } from './interfaces';
```

Comment:
Suggestion: use the full name instead the shortened one or a more descriptive alias.

I think shortening the interface name to a single character makes reading the code harder.

Reply:
I would prefer to omit the namespace name completely when using PropertyFilterProps in property filter code. However, I am not aware if that is possible at all. Btw, TS explicitly recommends using modules over namespaces ([see](https://www.typescriptlang.org/docs/handbook/namespaces-and-modules.html)).

Aliasing the namespace name to a single letter is a way to imitate the direct interface imports from a module.

I do agree that having the alias requires the code reader to acknowledge it first, but in the same time it reduces the clutter significantly. Wdyt?

Reply:
Actually, I found a better way -- updated the PR, please take a look.